redash/destinations/hangoutschat.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import requests
+
+from redash.destinations import *
+from redash.utils import json_dumps
+
+
+class HangoutsChat(BaseDestination):
+    @classmethod
+    def name(cls):
+        return "Google Hangouts Chat"
+    
+    @classmethod
+    def type(cls):
+        return "hangouts_chat"
```

Comment:
@pieter-venter yes, your current database records reference the old type (`hangoutschat`) which can no longer be found. The following query should fix this:

```sql
UPDATE notification_destinations SET type = 'hangouts_chat' where type = 'hangoutschat';
```

But if it creates too much hassle for you, we can revert back to `hangoutschat`. It just triggered my OCD :D

Reply:
OK great thanks. I did coincidentally found this myself too and fixed it using PGAdmin. Still learning the internals of Redash.

Honestly I don't mind. I went with one word, because I saw Mattermost, Pagerduty etc all used one word - but then again - their names are actually one word.

Let's go with your suggestion. I'll change the icon name to match the underscore and submit.